components/layouts/basic-content/two-columns--image-text.hbs

```diff
@@ -0,0 +1,45 @@
+<div class="section--bg-white">
+  <div class="basic-content--img-two-cols section__inner">
+
+    <div class="basic-content__img">
+      <picture src="https://via.placeholder.com/625x938" alt="ALT TEXT GOES HERE">
+        <source media="(min-width: 866px)" srcset="https://via.placeholder.com/625x938">
+        <source media="(min-width: 0px)" srcset="https://via.placeholder.com/640x360">
+        <img src="https://via.placeholder.com/625x938" alt="ALT TEXT GOES HERE">
+    </div>
+
+    <div class="basic-content__card">
+      <h3 class="basic-content__title">Flexible</h3>
+      <div class="basic-content__text">
+        <p>And we mean really flexible. Why be contained to a traditional career path, the four walls of an office, or
+          the typical 9-5?</p>
+      </div>
+    </div>
+
+    <div class="basic-content__card">
+      <h3 class="basic-content__title">Empowering</h3>
+      <div class="basic-content__text">
+        <p>Your career. Your call. We give you the opportunities, tools, and guidance to create the career you want.</p>
+      </div>
+    </div>
+
+    <div class="basic-content__card">
+      <h3 class="basic-content__title">Inclusive</h3>
+      <div class="basic-content__text">
+        <p>We’re all in when it comes to diversity. We’re constantly working to build a more inclusive environment where
+          everyone can contribute, grow, thrive—and express themselves.</p>
+        <p>Learn more about <a href="#">Equity &amp; Inclusion</a> at Vitamin T.</p>
+      </div>
+    </div>
+
+
+    <div class="basic-content__card">
+      <h3 class="basic-content__title">Equal</h3>
+      <div class="basic-content__text">
+        <p>Whether you’re a full-time staffer or a part-time freelancer, we’ve got your back with great benefits and HR
+          support.</p>
+      </div>
```

Comment:
Instead of each one of these being a content card, envision this as one blob output from the WYSIWYG, and let's just wrap it in something like `columns__right`